packages/headless/src/features/pagination/pagination-slice.test.ts

```diff
@@ -145,4 +146,36 @@ describe('pagination slice', () => {
     expect(nextState.firstResult).toEqual(123);
     expect(nextState.numberOfResults).toEqual(456);
   });
+
+  describe('#restoreSearchParameters', () => {
```

Comment:
Same comment here
```
beforeEach(() => {
  state.firstResult = 1;
  state.numberOfResults = 1;
})
```

Reply:
My guess is in both cases, he wants to test those parameters separately, so they don't influence each other. This is good practice generally for unit tests 😃